.pre-commit-config.yaml

```diff
@@ -45,86 +48,26 @@ repos:
   - id: check-yaml
 
 - repo: https://github.com/astral-sh/ruff-pre-commit
-  ##[[[cog
-  ## import re
-  ## version = re.search('ruff==([0-9\.]*)', open("constraints.txt").read())[1]
-  ## print(f"rev: v{version}")
-  ##]]]
   rev: v0.8.6
-  ##[[[end]]]
   hooks:
-    # Run the linter.
-    # TODO: include tests here
   - id: ruff
     files: ^src/
     args: [--fix]
-    # Run the formatter.
+    # TODO: include `tests` here
```

Comment:
highlighting a TODO

Reply:
I would postpone it for a subsequent PR.